### ISSUE TITLE
Add biome restriction to portal spawn locations

### ISSUE THREAD

**Issue by LordPINE** [YUNG-GANG/YUNGs-Better-Portals#6]
Would it be possible to add a biome or biome tag restriction to the locations the portals spawn? As I'd like to make a portal to the abyss dimension spawn in ocean biomes specifically, as that makes the most sense.

**Comment by bessiq** [comment]
Yes, this would be cool, especially considering the potential to have this mod work for more than just the Nether. (I've already created a rift that spawns in The End and teleports me to a mining dimension, but it would be nice if I could exclude the center island for instance).

Perhaps an implementation similar to how it's done with `"biomeTags"` in Better Mineshafts' variants file?

Really cool mod with awesome possibilities regardless. Thanks!
